src/routes/AppRoutes.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import HomePage from '../views/HomePage';
+import LoginPage from '../views/LoginPage';
+import Header from '../components/Header';
+import NotFoundPage from '../views/NotFoundPage';
+
+const AppRoutes: React.FC = () => {
+    return (
+        <Router>
+            <Header />
+            <Routes>
+                <Route path="/" element={<HomePage />} />
+                <Route path="/login" element={<LoginPage />} />
+                <Route path="*" element={<NotFoundPage />} />
+            </Routes>
+        </Router>
```

Comment:
This is good but there's a new way for routing which separates the main app from routes. It also has features like data loading, deferred loading out of the box, etc... look into it, hint: it uses createBrowserRouter and createRoutesFromElements